src/main/resources/assets/extended_industrialization/lang/ru_ru.json

```
@@ -0,0 +1,322 @@
+  "block.extended_industrialization.ev_tesla_receiver_hatch": "Приёмный люк Теслы экстремального напряжения",
+  "block.extended_industrialization.honey": "Мёд",
+  "block.extended_industrialization.hv_solar_panel": "Солнечная панель высокого напряжения",
+  "block.extended_industrialization.hv_tesla_receiver_hatch": "Приёмный люк Теслы высокого напряжения",
+  "block.extended_industrialization.large_configurable_chest": "Большой настраиваемый сундук",
+  "block.extended_industrialization.large_electric_furnace": "Большая электрическая печь",
+  "block.extended_industrialization.large_electric_macerator": "Большой электрический дробитель",
+  "block.extended_industrialization.large_steam_furnace": "Большая паровая печь",
+  "block.extended_industrialization.large_steam_macerator": "Большой паровой дробитель",
+  "block.extended_industrialization.lethal_tesla_coil": "Смертельная катушка Теслы",
+  "block.extended_industrialization.lv_solar_panel": "Солнечная панель низкого напряжения",
+  "block.extended_industrialization.lv_tesla_receiver_hatch": "Приёмный люк Теслы низкого напряжения",
+  "block.extended_industrialization.machine_chainer": "Соединитель машин",
+  "block.extended_industrialization.machine_chainer_relay": "Реле соединителя машин",
+  "block.extended_industrialization.manure": "Навоз",
+  "block.extended_industrialization.mv_solar_panel": "Солнечная панель среднего напряжения",
+  "block.extended_industrialization.mv_tesla_receiver_hatch": "Приёмный люк Теслы среднего напряжения",
+  "block.extended_industrialization.npk_fertilizer": "Удобрение NPK",
+  "block.extended_industrialization.phosphoric_acid": "Фосфорная кислота",
+  "block.extended_industrialization.polished_silver_machine_casing": "Полированный серебряный корпус машины",
+  "block.extended_industrialization.potassium_chloride": "Хлорид калия",
+  "block.extended_industrialization.potassium_hydroxide": "Гидроксид калия",
+  "block.extended_industrialization.processing_array": "Обрабатывающий массив",
+  "block.extended_industrialization.steam_farmer": "Паровой фермер",
+  "block.extended_industrialization.steel_alloy_smelter": "Стальная печь сплавов",
+  "block.extended_industrialization.steel_bending_machine": "Стальной листогиб",
+  "block.extended_industrialization.steel_brewery": "Стальная пивоварня",
+  "block.extended_industrialization.steel_canning_machine": "Стальной консервный аппарат",
+  "block.extended_industrialization.steel_composter": "Стальной компостер",
+  "block.extended_industrialization.steel_honey_extractor": "Стальной экстрактор мёда",
+  "block.extended_industrialization.steel_plated_bricks": "Кирпичи в стальной обшивке",
+  "block.extended_industrialization.steel_solar_boiler": "Стальной солнечный котёл",
+  "block.extended_industrialization.steel_waste_collector": "Стальной сборщик отходов",
+  "block.extended_industrialization.superconductor_tesla_receiver_hatch": "Приёмный люк Теслы SV",
```

SV = сверхН, исходя из [MI](https://github.com/AztechMC/Modern-Industrialization/blob/1.21.x/src/main/resources/assets/modern_industrialization/lang/ru_ru.json#L243)
```suggestion
  "block.extended_industrialization.superconductor_tesla_receiver_hatch": "Приёмный шлюз Теслы сверхН",
```